packages/dashmate/src/listr/tasks/setup/setupLocalPresetTaskFactory.js

```diff
@@ -228,23 +228,19 @@ function setupLocalPresetTaskFactory(
                   config.set('platform.drive.tenderdash.mode', 'validator');
 
                   // Setup logs
+                  const drivePrettyFilePath = homeDir.joinPath('logs', config.getName(), 'drive-pretty.log');
+                  config.set('platform.drive.abci.log.prettyFile.path', drivePrettyFilePath);
+
+                  const driveJsonFilePath = homeDir.joinPath('logs', config.getName(), 'drive-json.log');
```

Comment:
Nope, I didn't test. In deployment tool we use external log rotate. Do you think we should expose flush command for Drive so log rotate will call it? Do you think it's a big deal? In worst case scenario we just add lines to new file instead of old one.

Reply:
See https://unix.stackexchange.com/questions/440004/why-is-kill-hup-used-in-logrotate-in-rhel-is-it-necessary-in-all-cases

Looks like with `logrotate`, we need to handle this signal a bit different - close and recreate the file; otherwise writes could still go to old file, regardless of the move done by logrotate.

Reply:
And your change might be a breaking change for `max_files > 0` - I mean, it can stop working. That's why I suggested this test.